src/snowflake/cli/api/cli_global_context.py

```diff
@@ -54,10 +54,11 @@ class _CliGlobalContextManager:
     project_env_overrides_args: dict[str, str] = field(default_factory=dict)
 
     # FIXME: this property only exists to help implement
-    # nativeapp_definition_v2_to_v1. Consider changing the way
-    # this calculation is provided to commands in order to remove
-    # this logic (then make project_definition a non-cloned @property)
+    # nativeapp_definition_v2_to_v1 and single_app_and_package.
+    # Consider changing the way this calculation is provided to commands
+    # in order to remove this logic (then make project_definition a non-cloned @property)
```

Comment:
I know this is proliferating an undesirable pattern, but I think it's worth it for the simplicity of the change. We'll have a better opportunity to fix this once this v1-to-v2 refactor is done.

Reply:
ok